index.html

```diff
@@ -264,8 +260,16 @@ <h2 id='accommodation'>Accommodation</h2>
                 <a href='https://corfuimperial.reserve-online.net/?checkin=20/05/2016&bkcode=SUMSCH'>book their rooms using the code SUMSCH</a> 
                 at the discounted rate of €150/night for double- and €130/night for single-occupancy (taxes and breakfast included). </p>
                 
+                <strong>Alternative options</strong>
+                
+                <a href='http://www.hotelnefeli.com/'>Hotel Nefeli</a> (20' walking from the venue) offers rooms iscounted rate 
+                of €42/night for double- and €34/night for single-occupancy (taxes and breakfast included). 
+                To reserve a room <a href='mailto:hotelnefeli@web.de'>email</a> them 
+                directly and mention that you are attending the cryptocurrencies event.
                 
-                We will soon provide more walking-distance accommodation options.</p>
+                For more options please check on 
+                <a href='http://www.booking.com/searchresults.html?label=gen173nr-1FCAEoggJCAlhYSDNiBW5vcmVmcgV1c19tYYgBAZgBMbgBD8gBDNgBAegBAfgBAqgCAw;sid=f0b991addcd69497b1462e09d88de50a;dcid=12;checkin_monthday=29&checkin_year_month=2016-5&checkout_monthday=3&checkout_year_month=2016-6&class_interval=1&dest_id=-821013&dest_type=city&dtdisc=0&group_adults=2&group_children=0&hlrd=0&hyb_red=0&inac=0&label_click=undef&nha_red=0&no_rooms=1&offset=0&postcard=0&redirected_from_city=0&redirected_from_landmark=0&redirected_from_region=0&review_score_group=empty&room1=A%2CA&sb_price_type=total&score_min=0&si=ai%2Cco%2Cci%2Cre%2Cdi&src=index&ss=Kommeno%2C%20Ionian%20Islands%2C%20Greece&ss_all=0&ss_raw=corfu%20komme&ssb=empty&sshis=0&tfl_cwh=1'>booking.com</a>.
```

Comment:
not sure how to fix this - I want the link to point directly to correct location/dates